### PR TITLE
chore: add sentry cli to push source map to sentry.io

### PR DESCRIPTION
**Pull request by tanphamhaiduong**
chore: add sentry cli to push source map to sentry.io

### CODE REVIEW

File: scripts/webpack/webpack.base.prod.js [hunk 1]
```diff
@@ -3,26 +3,23 @@ const FaviconsWebpackPlugin = require('favicons-webpack-plugin')
...
 const { GenerateSW } = require('workbox-webpack-plugin')
-const HashedModuleIdsPlugin = require('webpack').HashedModuleIdsPlugin
-const { EnvironmentPlugin } = require('webpack')
-const { execSync } = require('child_process')
-// const SentryWebpackPlugin = require('@sentry/webpack-plugin')
-const { PATHS } = require('./constants')
-const { getVersionTag } = require('../release/utils')
-const hashFiles = require('../utils/hash-files')
 const path = require('path')
+const SentryWebpackPlugin = require('@sentry/webpack-plugin')
```
Comment (vuhuucuong) [inline scripts/webpack/webpack.base.prod.js:7]: So we're adding sentry source map to `webpack.base.prod.js`, does it mean that when we `yarn:build` in local machine, source map is still pushed to sentry?
Can we only do this when `CI = true` ?https://github.com/reapit/foundations/blob/master/.github/workflows/release-develop.yml#L16

Comment (tanphamhaiduong) [reply scripts/webpack/webpack.base.prod.js:7]: @vuhuucuong that's my concern, so let me find the way to remove build from PRs, and only accept in release develop and production.

Comment (willmcvay) [reply scripts/webpack/webpack.base.prod.js:7]: Good spot @vuhuucuong

File: scripts/webpack/webpack.base.prod.js [hunk 2]
```diff
@@ -184,3 +176,20 @@ module.exports = {
     modules: false,
   },
 }
+
+if (process.env.IS_RELEASE) {
```
Comment (willmcvay) [inline scripts/webpack/webpack.base.prod.js:180]: 👍

**Pull request merged**